### PR TITLE
Schema.sql and more db tasks (db:create, db:drop, db:schema:load, db:schema:dump)

### PR DESCRIPTION
**Pull request by mfine**
Bring in a `schema.sql` file to capture schema. Add tasks for creating and dropping a database, plus schema loading and dumping. Should facilitate a bootstrap command. Definitely up to making this cleaner.

### CODE REVIEW

**Comment by mfine** [comment]
@brandur @pedro think this is all set to go - what do you guys think?

**Comment by mfine** [comment]
I stripped down the db.rake file some more.

File: lib/tasks/db.rake [hunk 1]
```diff
@@ -1,34 +1,68 @@
+require "uri"
+require "sequel"
```
Comment (pedro) [inline lib/tasks/db.rake:2]: I think we can take this out, the tasks below depends on the `:env` task, which loads Bundler.

**Comment by pedro** [comment]
This looks awesome to me! I want to make these db tasks work both with test and dev dbs but can address this later.

File: lib/tasks/db.rake (hunk 1 above)
Comment (mfine) [inline lib/tasks/db.rake:2]: If I take those out, shit breaks!

**Comment by mfine** [comment]
Going to bring this in - then can start evolving patterns here.

**Comment by brandur** [comment]
This is seriously awesome. +ONE!

**Pull request merged**